controller/controller_server/csi_controller_server.py

```diff
@@ -23,6 +22,7 @@
 from controller.csi_general import csi_pb2_grpc
 
 logger = None  # is set in ControllerServicer::__init__
+logger: logging.Logger
```

Comment:
How this change is related to this PR?
I'm not sure we need it.

Reply:
it's not, it only allows auto-completion for `logger`.
do you think it should be undone?

Reply:
I think its good that it no longer shows warnings

Reply:
I'm not sure why we use the `logging.Logger` class instead of using `logging`.
We still see a warning in case of:
`logger = None  # is set in ControllerServicer::__init__`
`logger: logging.Logger`
since `None` is not an instance of `logging.Logger`

[This](https://github.com/zingero/smart-fridge/blob/ef530fd0a64e7fd88f570a519ac505d2788039c7/main.py) is an example (from a different repository) of using the `logging` module directly.

Reply:
I wouldn't worry about this warning (a lot of them don't mean much to us), but feel free to open a ticket for it

Reply:
CSI-2894

Reply:
@zingero can this conversation be resolved?